modules/shared/partials/acid-transactions.adoc

```diff
@@ -1,84 +1,101 @@
 = ACID Transactions
 
 
+//-------------------------- HOWTO PARTIALS ------------------------
 
 // tag::intro[]
-This document presents a practical HOWTO on using Couchbase transactions, following on from our xref:7.1@server:learn:data/transactions.adoc[transactions documentation].
+Couchbase Distributed xref:7.1@server:learn:data/transactions.adoc#overview[ACID (atomic, consistent, isolated, and durable)] Transactions allow applications to perform a series of database operations as a single unit -- meaning operations are either committed together or all undone.
+Transactions are distributed and work across multiple documents, scopes, and collections, which can reside on multiple nodes.
 // end::intro[]
 
 
-
-
 // tag::requirements[]
-* NTP should be configured so nodes of the Couchbase cluster are in sync with time.
-* The application, if it is using xref:concept-docs:xattr.adoc[extended attributes (XATTRs)], must avoid using the XATTR field `txn`, which is reserved for Couchbase use.
```

Comment:
Is there a reason to ditch these two?  The second is a real edge case, but NTP is v important.

Reply:
I just pulled these out of common, and decided to put them in the individual SDKs, so this is still documented.